flexmeasures/ui/templates/views/new_dashboard.html

```diff
@@ -138,37 +126,16 @@
         .bindPopup(`
                             <div class="leaflet-marker-popup">
                                 <div class="row">
-                                    <div class="col-sm-12">
+                                    <div class="col-sm-9">
```

Comment:
As it sits now, the space isn't being used. The title uses 9 cols, and the button below uses 3.
It leads to a lot of empty space on the right side.

Was that the intention?

And what would it take to display the current state in text form, as we discussed, e.g. "Current state: X% (of <capacity> <unit>".

Of course, the latter could also be done in a new ticket. But if it's small, this can be a good place as we are now taking that feature away. I might volunteer.